src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java

```diff
@@ -81,8 +81,16 @@
 	private ArrayList<Plankton> plankton = new ArrayList<Plankton>();
 
 	// Images in the game
-	private URL reefURL = getClass().getResource("/resources/CoralReef.jpg");
-	private Image reef = new ImageIcon(reefURL).getImage();
+    //difficulty1
```

Comment:
Can you use tabs for indentation to match the original file?  It seems to use one tab per level, while you're using four spaces.